app/src/androidTest/java/com/mapbox/maps/testapp/annotation/generated/PointAnnotationManagerAndroidTest.kt

```diff
@@ -129,6 +129,14 @@ class PointAnnotationManagerAndroidTest : BaseMapTest() {
     assertEquals(testValue, pointAnnotationManager.symbolSpacing)
   }
 
+  @Test
+  fun testSymbolZOrder() {
+    val testValue = SymbolZOrder.AUTO
```

Comment:
nit:
```suggestion
    val expectedValue = SymbolZOrder.AUTO
```

Reply:
This is generated code and changing it will affect all other tests but I agree with @alexander-kulikovskii that all `testValue` variables need to be rename to `expectedValue`.